index.js

```diff
@@ -330,6 +330,36 @@ walletRouter.post('/melt-tokens', (req, res) => {
   }
 });
 
+/**
+ * GET request to get utxo details
+ * For the docs, see api-docs.js
+ */
+walletRouter.get('/utxo-details', (req, res) => {
+  const wallet = req.wallet;
+  const options = req.query || {};
+  const utxoDetails = wallet.getUtxos(options);
```

Comment:
Shouldn't we paginate this response? I feel a larger wallet can get hundreds of thousands of utxos and we don't want to return all of them.

Reply:
I added a `TODO` regarding this scenario. Do you think that is enough for now?

Reply:
I think a `TODO` is enough for now. It's possible that we have a wallet with hundreds of thousands of utxos but really unlikely. After the wallet-service integration it can be implemented.

Reply:
I feel we should put a limit and flag if the limit has been reached. I don't feel it is reasonable to return 1 million utxos in any case. Maybe a limit of 10,000 utxos? And the `TODO` as well, of course.

Reply:
@msbrogli, after reviewing the filtering behavior, I noted that `max_utxos` may already serve as limiting because it defines the number of returned `utxos` and it has a default value of 255. So, even if the request does not specify `max_utxos`, it will return only 255 `utxos`. Even with `max_utxos`, should I add a new limit flag?

Reply:
You're right. It will return at most 255 `utxos` unless the user increases the `max_utxos` argument. Well, as it is already protected, I feel we can add pagination later.